docs/demo/gaiden-css/components/list/demo.html

```diff
@@ -15,7 +15,7 @@
 <body>
   <article>
     <header>
-      <h5>Topic List Header</h5>
+      <h5 class="list__title">Topic List Header</h5>
```

Comment:
`list__title` outside a block? BEM don't like this :/

Reply:
Our component is built wrong. For inside a list (<ul>) we can not have another item not your own <li> this title should go from an element that was default to use on any component that needs it, without having it attached to the list .

What do you think?

Reply:
The best scenario for this composition would be:

```
<wrapper>
  <title-component>
  <list-component>
```

But the title component is much generic, what do you think `title-list`?

Reply:
@williamcosta we discuss this approach in person. I like the idea of generic component, uncoupled with markup. But we can use same things like:

```HTML
<ul class='list'>
  <li class='list__item list__item--with-header'>
    List Header Title
    <ul class='list list--sub'>
      <li class='list__item'>List Item</li>
      <li class='list__item'>List Item</li>
    </ul>
  </li>
</ul>
```

Or something like:

```HTML
<ul class='list'>
  <li class='list__item'>
    <span class='list__item--header'>List Header Title</span>
    <ul> <!-- and here, we specify what markup element can be a sub list (ul, ol, dd) -->
      <li class='list__item'>List Item</li>
      <li class='list__item'>List Item</li>
    </ul>
  </li>
</ul>
```